src/Faker/Provider/Internet.php

```diff
@@ -125,13 +125,12 @@ public static function toAscii($string)
             'ო' => 'o','პ' => 'p','ჟ' => 'z','რ' => 'r','ს' => 's','ტ' => 't',
             'უ' => 'u','ფ' => 'p','ქ' => 'k','ღ' => 'g','ყ' => 'q','შ' => 's',
             'ჩ' => 'c','ც' => 't','ძ' => 'd','წ' => 't','ჭ' => 'c','ხ' => 'k',
-            'ჯ' => 'j','ჰ' => 'h','ā' => 'a','ţ' => 't','ʼ' => "'", '̧' => '',
```

Comment:
And I think hole this array should be protected static var, and in method ```toAscii``` there should be only logic.